frontend/src/components/Search.js

```diff
@@ -1,15 +1,19 @@
 import React, { useState, useEffect } from 'react'
-import { useHistory } from 'react-router-dom'
+import { useHistory, useLocation } from 'react-router-dom'
 import { useDispatch } from 'react-redux'
 import { saveSearchedPosts, saveSearchedQuery } from '../store/actions/actions'
 import { useOnRouteLeave } from '../utils/customHooks/useOnRouteLeave'
 import { ReactComponent as SearchIcon } from '../assets/icons/search.svg'
+import { toast } from 'react-toastify'
+import 'react-toastify/dist/ReactToastify.css'
 
 const Search = () => {
   const [input, setInput] = useState('')
   const [timeoutID, setTimeoutID] = useState(null)
+  const [disableInput, setDisableInput] = useState(false)
```

Comment:
`disableInput` sounds more like a function name, but it is a boolean. 
`isInputDisabled` is better as it clearly says what it is.

Reply:
Yea, you're right